hack/generator/pkg/astmodel/storage_conversion_context.go

```diff
@@ -42,3 +50,27 @@ func (c *StorageConversionContext) ResolveEnum(t Type) (TypeName, *EnumType, boo
 
 	return name, enumType, true
 }
+
+// ResolveObject takes a Type and resolves it into the name of an object type,
+// returning true if found or false if not.
+func (c *StorageConversionContext) ResolveObject(t Type) (TypeName, bool) {
```

Comment:
This name confused me - I expected it to take a TypeName and return the object type, but this returns the name after checking that the referent type is an object type. Maybe call it `ObjectTypeName`?

Reply:
Good point. This started as a similar method to `ResolveEnum` but was returning values never used, so I simplified it, but without reconsidering the name.

Starting to suspect it's now the wrong abstraction as I'm having trouble naming it. 🤔